garage/tf/models/continuous_mlp_model.py

```diff
@@ -0,0 +1,70 @@
+"""Discrete MLP Model."""
```

Comment:
Should this be Continuous MLP Model?

Reply:
it would be nice of these docstrings actually described the model rather than just rehashing its name

It could just as easily read `"""A model composed only of a multi-layer perceptron (MLP), which maps real-valued inputs to real-valued outputs."""`